A3A/addons/core/Params.hpp

```diff
@@ -341,8 +341,8 @@ class Params
     class lootCrateDistance
     {
         title = $STR_params_lootCrateDistance;
-        values[] = {10, 25, 50, 75, 100};
-        texts[] = {"10", "25", "50", "75", "100"};
+        values[] = {10, 25, 50, 75, 100, "200", "300", "400"};
```

Comment:
Values shouldn't be a string
```suggestion
        values[] = {10, 25, 50, 75, 100, 200, 300, 400};
```

Reply:
i feel like an idiot im sorry  it was pretty late when i did this PR

Reply:
> i feel like an idiot im sorry it was pretty late when i did this PR

Everyone makes mistakes, that's why we review changes first

You can either commit the suggestion or push a commit yourself to do the required changes